core-java-modules/core-java-string-conversions-2/src/test/java/com/baeldung/stringtocamelcase/StringtocamelcaseApplicationTests.java

```diff
@@ -0,0 +1,61 @@
+package com.baeldung.stringtocamelcase;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class StringtocamelcaseApplicationTests {
```

Comment:
Let's have the name of this in camel case too. And it needs to be suffixed `UnitTest` to pass the jenkins build